src/main/java/bio/terra/cli/service/GoogleOauth.java

```diff
@@ -261,4 +265,26 @@ public static AccessToken getAccessToken(GoogleCredentials credential) {
     }
     return credential.getAccessToken();
   }
+
+  /**
+   * Revoke token (https://developers.google.com/identity/protocols/oauth2/web-server#tokenrevoke).
+   *
+   * <p>Google Java OAuth library doesn't support revoking tokens
+   * (https://github.com/googleapis/google-oauth-java-client/issues/250), so make the call ourself.
+   *
+   * @param credential credentials object
+   */
+  public static void revokeToken(GoogleCredentials credential) {
+    String endpoint = "https://oauth2.googleapis.com/revoke";
+    Map<String, String> headers =
+        ImmutableMap.of("Content-type", "application/x-www-form-urlencoded");
+    Map<String, String> params =
+        ImmutableMap.of("token", credential.getAccessToken().getTokenValue());
+
+    try {
+      HttpUtils.sendHttpRequest("https://oauth2.googleapis.com/revoke", "POST", headers, params);
+    } catch (IOException ioEx) {
+      throw new SystemException("Unable to revoke token");
```

Comment:
nit: wrap the `ioEx` in the `SystemException`

Reply:
Done